libraries/testing/tester.cpp

```diff
@@ -353,6 +353,9 @@ namespace eosio::testing {
           }
       });
 
+      control->disable_async_voting(true);      // vote synchronously so we don't have to wait for votes
```

Comment:
Should this be set to `false` by default? Only let specific tests (like fork tests) set it to `true` when needed?

Reply:
I think it is better to have async voting disabled by default. I don't see why any test would need asynchronous voting.

Reply:
Asynchronous voting is used in production. It is better to leave some tests exercising it.